packages/asset-uploader/src/utils/format-asset.test.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { Asset, Location } from "@webstudio-is/prisma-client";
+
+const commonAsset: Asset = {
+  id: "sa-546",
+  name: "test",
+  location: Location.FS,
+  projectId: "id",
+  size: 2135,
+  format: "png",
+  createdAt: new Date(),
+  description: "",
+  meta: JSON.stringify({ width: 200, height: 200 }),
+};
+
+describe("getAssetPath", () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = {
+      ...OLD_ENV,
+    };
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV;
+  });
+
+  describe("local", () => {
+    test("returns none on Location.FS if VERCEL_ENV not set", () => {
+      process.env.VERCEL_ENV = undefined;
```

Comment:
do you like the mocking more than passing env as a second argument that defaults to process.env? It's clean and doesn't depend on jest magic